pkg/codegen/testing/test/testdata/simple-enum-schema/dotnet/Enums.cs

```diff
@@ -36,6 +36,7 @@ private CloudAuditOptionsLogName(string value)
         /// What if triple quotes """ are used in the description
         /// </summary>
         public static CloudAuditOptionsLogName Synthetic { get; } = new CloudAuditOptionsLogName("SYNTHETIC");
+        public static CloudAuditOptionsLogName CloudAuditOptionsLogName_NO_NAME { get; } = new CloudAuditOptionsLogName("_NO_NAME");
```

Comment:
Why do we need the whole enum type name here? Identifiers are ok to start with underscore.

Reply:
Same as https://github.com/pulumi/pulumi/pull/9453#discussion_r857810830

> I don't know. My suspicion is that it provides a way to guarantee uniqueness for when enums don't have unique namespaces. I don't know why this feature is present in nodejs. I'll make a note to clean this up for 4.0.